src/app/frontend/deploy/createsecret.html

```diff
@@ -0,0 +1,64 @@
+<!--
+Copyright 2015 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<md-dialog aria-label="Create a new secret" layout="column" layout-padding width="500px">
+  <md-content layout-padding>
+    <h4 class="md-title">Create a new secret</h4>
+    <p>The new secret will be added to the cluster.</p>
+
+    <form name="ctrl.secretForm" ng-submit="ctrl.createSecret()" novalidate>
+      <kd-help-section>
+        <md-input-container class="md-block">
+          <label>Secret name</label>
+          <input name="secretName" ng-model="ctrl.secretName" md-maxlength="{{ctrl.secretNameMaxLength}}" ng-pattern="ctrl.secretNamePattern" required>
+          <div ng-messages="ctrl.secretForm.secretName.$error">
+            <div ng-message="pattern">Name must be alphanumeric, followin a DNS syntax (e.g. new.image-pull.secret)</div>
+            <div ng-message="md-maxlength">Name is too long</div>
+            <div ng-message="required">Name is required</div>
+          </div>
+        </md-input-container>
+        <kd-user-help>
+            A secret with the specified name will be created in the namespace <b>{{ctrl.namespace}}</b>.
```

Comment:
Dont use <b>. This is very strong. Apply a class and use font weight from _variables.scss.